cmd/ci-images-mirror/main.go

```diff
@@ -306,13 +302,21 @@ func main() {
 		if err := quayiociimagesdistributor.RegisterMetrics(); err != nil {
 			logrus.WithError(err).Fatal("failed to register metrics")
 		}
+		ignoreImageStreamTags := sets.New[string](opts.quayIOCIImagesDistributorOptions.ignoreImageStreamTagsRaw.Strings()...)
+		if opts.config != nil {
+			for k := range opts.config.SupplementalCIImages {
+				logrus.WithField("target", k).Debug("Ignore target of supplemental CI images on mirroring")
+				ignoreImageStreamTags.Insert(k)
+			}
+		}
+		logrus.WithField("tags", ignoreImageStreamTags.UnsortedList()).Infof("%s will ignore mirroring those tags", quayiociimagesdistributor.ControllerName)
```

Comment:
This might be incorrect.

Reply:
It should be `Info`, a typo I suppose

Reply:
```
Infof("%s...
```
the `f` is there because i used the format string `%s`.

Reply:
makes sense.

Reply:
Maybe using withField and %s together is a bad styling but it is fine syntactically.